server/models/user.model.js

```diff
@@ -1,29 +1,35 @@
-/**
- * User Schema
- */
-module.exports = (sequelize, DataTypes) => {
-    const User = sequelize.define('User', {
-        id: {
-            type: DataTypes.INTEGER,
-            autoIncrement: true,
-            primaryKey: true,
+import { Schema } from 'mongoose';
+import mongoConn from '../../config/mongo';
+
+const UserSchema = new Schema({
+    uuid: {
+        type: String,
+        required: true,
+        unique: true,
+        index: true,
+    },
+    username: {
+        type: String,
+        required: true,
+    },
+    devices: [new Schema({
```

Comment:
Is there a reason this is a new schema?

Reply:
Thought it was necessary - just tested and it's not! I'll remove it